scenarios/steadystate.profile.yml

```diff
@@ -0,0 +1,43 @@
+# This file adds a new "Steady State" results (working set and cpu) to the "application" job.
+# It uses all the measurements after the warmup is done.
+
+results:
+
+- name: benchmarks/working-set/steady
+  description: Steady State Working Set (MB)
```

Comment:
Do we really want to call this "Steady State"? Would it be better to call it "Load Working Set P90 (MB)"?

Reply:
"Load Working Set P90 (MB)" sounds like a fancy car name. Much better.